.i18nrc.json

```diff
@@ -43,7 +42,7 @@
     "esQuery": "packages/kbn-es-query/src",
     "presentationUtil": "src/plugins/presentation_util",
     "indexPatternEditor": "src/plugins/index_pattern_editor",
-    "indexPatternFieldEditor": "src/plugins/index_pattern_field_editor",
+    "indexPatternFieldEditor": "src/plugins/data_view_field_editor",
```

Comment:
We don't change prefixes of i18n keys?

Reply:
We can, but I preferred to limit the number of changes in this PR.